src/ResourceManager/OperationalInsights/Commands.OperationalInsights.Test/ScenarioTests/DataSourceTests.ps1

```diff
@@ -0,0 +1,174 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Create, update, and delete data sources
+#>
+function Test-DataSourceCreateUpdateDelete
+{
+    $wsname = Get-ResourceName
+    $dsName = Get-ResourceName
+    $rgname = Get-ResourceGroupName
+	$subId1 = "0b88dfdb-55b3-4fb0-b474-5b6dcbe6b2ef"
```

Comment:
@haidaocht Please format this document properly. replace all TAB symbols with 4 space symbols